chrome-launcher/chrome-launcher.ts

```diff
@@ -94,14 +96,15 @@ export class Launcher {
     this.chromeFlags = defaults(this.opts.chromeFlags, []);
     this.requestedPort = defaults(this.opts.port, 0);
     this.chromePath = this.opts.chromePath;
+    this.enableExtensions = defaults(this.opts.enableExtensions, false);
   }
 
   private get flags() {
     const flags = DEFAULT_FLAGS.concat([
       `--remote-debugging-port=${this.port}`,
       // Place Chrome profile in a custom location we'll rm -rf later
       `--user-data-dir=${this.userDataDir}`
-    ]);
+    ]).filter(flag => this.enableExtensions ? flag !== '--disable-extensions' : true);
```

Comment:
this might be clearer as a separate conditional rather than nested in the `flags` declaration. e.g.

```js
if (this.enableExtensions) {
  flags.filter(flag => flag !== '--disable-extensions');
}
```